src/Core/src/Platform/Windows/ButtonExtensions.cs

```diff
@@ -107,27 +107,32 @@ public static void UpdateTextColor(this ButtonBase platformButton, ITextStyle bu
 		}
 
 		public static void UpdateTextColor(this ButtonBase platformButton, Color textColor)
+		{
+			platformButton.UpdateTextColor(textColor, TextColorResourceKeys);
+		}
+
+		public static void UpdateTextColor(this ButtonBase platformButton, Color textColor, string[] resourceKeys)
```

Comment:
Can we leave this `internal` ? I don't think we need to make this public.

Reply:
Changed, could you review it again?. Thnx!